lib/commons/utils/index.js

```diff
@@ -1,10 +0,0 @@
-/* exported utils */
-/* global axe */
-/*eslint no-unused-vars: 0*/
-/**
- * Namespace for general utilities.
- * @namespace utils
- * @memberof axe.commons
- */
-
-var utils = (commons.utils = axe.utils);
```

Comment:
We need to keep this around. Removing this would be a breaking change.

Reply:
What would this break?

Reply:
Commons are pseudo-public. They are used in custom rules which clients use. We have to be careful about changing commons.